### ISSUE TITLE
Does not report any errors for some invalid pages

### ISSUE THREAD

**Issue by tkrotoff** [spk/validate-website#11]
Small test with HTML containing errors: http://www.cs.sfu.ca/~ggbaker/165-draft/files/invalid.html

Result:
```Shell
bundle exec validate-website --site 'http://www.cs.sfu.ca/~ggbaker/165-draft/files/invalid.html' --verbose --markup --not-found
validating http://www.cs.sfu.ca/~ggbaker/165-draft/files/invalid.html
1 visited, 0 failures, 0 not founds, 0 errors
```

On validator.nu: https://html5.validator.nu/?doc=http%3A%2F%2Fwww.cs.sfu.ca%2F~ggbaker%2F165-draft%2Ffiles%2Finvalid.html

```
1. Error: End tag a violates nesting rules.
From line 11, column 127; to line 11, column 130
rtium">W3C</a>, the 
2. Error: End of file reached when inside an attribute value. Ignoring tag.
At line 17, column 7
↩</body>↩</html>
```

Edit:
For www.w3.org, validate-website reports errors, so the problem is more subtil.
On my Jekyll website, I've added errors to my pages => no report (validate-website with localhost:4000 or validate-website-static, btw I didn't get the difference between validate-website and validate-website-static)

**Comment by spk** [comment]
Thanks for your report. It was a redirection (301) and with this [host](https://github.com/spk/validate-website/commit/ea93f10606e6d0717893613b1a93f25e565205d2) it's working again.
I would like to improve that with #8 .

**Comment by spk** [comment]
Thanks :exclamation: 
https://rubygems.org/gems/validate-website/versions/1.0.1

**Comment by spk** [comment]
https://github.com/spk/validate-website/releases/tag/v1.0.5
